pandas/tools/pivot.py

```diff
@@ -170,6 +170,9 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
     if len(index) == 0 and len(columns) > 0:
         table = table.T
 
+    if isinstance(table, Series):
```

Comment:
actually I think a you can simply always make ``aggfunc`` a list (if its not None) and this will just work.